### PR TITLE
Define the endpoints as a list

### LINKED ISSUE #126: Define the endpoints as a list
> > Is there a convention for how the ends are referenced? This has a mix of 'one' and 'other' with '-1' and '-2'
> 
> No change made. I think it's reasonable to have -1 and -2 in the model, but 'one' and 'other' in description. Tweak my English...
> 
> _Originally posted by @haomianzheng in https://github.com/haomianzheng/IETF-ACTN-YANG-Model/issues/125#issuecomment-1356081448_

It may be better (and future proof) to define a list of endpoints with two and only two elements:

```
       +--rw endpoint [id]
          +--rw id     string
          +--rw uni    -> /l1-connectivity/access/unis/uni/id
```

### PR DESCRIPTION
**Pull request by italobusi**
Fix #126

### CODE REVIEW

File: YANG/ccamp/l1csm/ietf-l1csm.yang [hunk 1]
```diff
@@ -106,46 +106,30 @@ module ietf-l1csm {
...
-        type string;
-        must '. != ../../endpoint-1/uni' {
-          error-message
-            "The two end point IDs must not be equal to each other. ";
+          "The list of the two of the subscriber L1VC end points";
+        leaf endpoint-id {
+          type string;
+          mandatory true;
+          description
+            "The subscriber L1VC end point ID of one end";
```
Comment (haomianzheng) [inline YANG/ccamp/l1csm/ietf-l1csm.yang:123]: OLD
The subscriber L1VC end point ID of one end
NEW
The subscriber end point IDs of the L1VC

File: YANG/ccamp/l1csm/ietf-l1csm.yang [hunk 2]
```diff
@@ -106,46 +106,30 @@ module ietf-l1csm {
...
-      leaf uni {
-        type leafref {
-          path "/l1-connectivity/access/unis/uni/uni-id";
+        leaf uni {
+          type leafref {
+            path "/l1-connectivity/access/unis/uni/uni-id";
+          }
+          mandatory true;
+          description
+            "This UNI supporting the subscriber L1VC end point.";
```
Comment (haomianzheng) [inline YANG/ccamp/l1csm/ietf-l1csm.yang:131]: OLD: 
This UNI supporting the subscriber L1VC end point
NEW
The UNI supporting the subscriber L1VC end point

File: YANG/ccamp/l1csm/ietf-l1csm.yang (hunk 1 above)
Comment (italobusi) [reply YANG/ccamp/l1csm/ietf-l1csm.yang:123]: Ok in principle. What about:

```suggestion
            "The subscriber L1VC end point ID";
```

File: YANG/ccamp/l1csm/ietf-l1csm.yang (hunk 2 above)
Comment (italobusi) [reply YANG/ccamp/l1csm/ietf-l1csm.yang:131]: Ok

```suggestion
            "The subscriber L1VC end point ID";
```

**Pull request merged**